controllers/index.js

```diff
@@ -136,4 +136,22 @@ module.exports = {
   },
 
   webhook: (req, res) => { },
+
+  wrappedGetTopArtists: async (req, res) => {
+    try {
+      const limit = req.query.limit || 3;
+      const year = req.query.year || '2020';
```

Comment:
Let's remove 2020 and make year a required parameter from Joi.

Reply:
Did it this way because I like the idea of having the first year as the default but sure.

Reply:
Done